tools/repo-utils/backport.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
```

Comment:
I tend to always use `set -euo pipefail` (IIRC, one fails on undefined variables, one makes the result of a pipe fail if any part fails, and I can't remember the other one). I think that'd be safer since there are some pipes in there.